### PR TITLE
CI: detect flakes due to packagemanifest not found and cluster unreachable 

### PR DESCRIPTION
**Pull request by kpouget**
* 83693b8 - CI: run: Remove old single-worded entrypoint definitions

---

* af9948c - CI: run: detect when cluster became unreachable during the testing

---

* 6c9b3d0 - toolbox: expose 'artifact_dir' as ansible variable

And ensure it is set in 'check_deps' role.

---

* 9a51d1a - gpu_operator_deploy_from_operatorhub: mark packagemanifests/gpu-operator-certified not found as flake

---

* 6d4b600 - nfd_operator_deploy_from_operatorhub: mark packagemanifests/nfd not found as flake

---

* dd01e42 - docs: introduce 'ci' section and document special files

---

### CODE REVIEW

File: docs/ci/intro.rst [hunk 1]
```diff
@@ -0,0 +1,20 @@
+============
+Introduction
+============
+
+As part of Red Hat PSAP team, we runs nightly a set of tests to
```
Comment (omertuc) [inline docs/ci/intro.rst:5]: `run`

File: docs/ci/intro.rst [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+validate the proper execution of the operators we are in charge of:
+
+- the `GPU Operator <https://openshift-psap.github.io/ci-dashboard/gpu-operator_daily-matrix.html>`_
+- the `Node Feature Discovery operator <https://openshift-psap.github.io/ci-dashboard/nfd_daily-matrix.html>`_
+- the `Node Tuning Operator <https://openshift-psap.github.io/ci-dashboard/nto_daily-matrix.html>`_
+- the `Special Resource Operator <https://openshift-psap.github.io/ci-dashboard/sro_daily-matrix.html>`_
+
+The execution of these nightly tests is controlled by the code int the
```
Comment (omertuc) [inline docs/ci/intro.rst:13]: in

File: docs/ci/intro.rst [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+`ci-artifacts <https://github.com/openshift-psap/ci-artifacts/>`_
+repository, and the test steps are driven by the `toolbox
+<https://openshift-psap.github.io/ci-artifacts/index.html#psap-toolbox>`_
+commands.
+
+The nightly test overview is generated with the help of `ci-dashboard
```
Comment (omertuc) [inline docs/ci/intro.rst:19]: of the

File: docs/ci/files.rst [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+            "unreachable": 0
+        }
+    },
+    "status": "finished"
+
+
+- ``$ARTIFACT_DIR/FAILURE``: the presence this file after the CI
+  execution indicates that the testing failed.
+
+- ``$ARTIFACT_DIR/FLAKE``: the presence of this file after a toolbox
```
Comment (omertuc) [inline docs/ci/files.rst:38]: Great idea

**Comment by kpouget** [comment]
> /lgtm other than a few typos
> /hold

thanks for the proof reading @omertuc , all comments have been addressed.
(I think the number of typo betrays that I forced myself into writing these documentation lines ;-)

**Comment by kpouget** [comment]
> /lgtm other than a few typos
> /hold

@omertuc good for merging this one?
all the comments (typos) have been addressed

**Pull request merged**